INN/umbrella-sfpublicpress
# Prefooter and footer

## Changes

This pull request makes the following changes:

- Adds styling for the prefooter
- Adds styling for the footer

Desktop:
![Screen Shot 2020-01-13 at 11 55 48 AM](https://user-images.githubusercontent.com/18353636/72275354-bb14fe00-35fb-11ea-9194-db29f52f7cb3.png)

Mobile:
![screencapture-sfpp-test-2020-01-13-11_55_55](https://user-images.githubusercontent.com/18353636/72275360-bea88500-35fb-11ea-8362-49a2675c2035.png)

## Why



For #24 and #25 

## Testing/Questions

Features that this PR affects:

- Prefooter
- Footer


Questions that need to be answered before merging:

- [ ] Does everything look ok?

Steps to test this PR:

1. Select `Include an additional widget region ("sidebar") just above the site footer region` in theme options
2. Select the 4 equal columns footer in theme options
3. In `Before Footer` widget area add the `Largo About Site` widget with the title `About Us`
4. In the `Before Footer` widget area add the `Largo Donate Widget` with no title or CTA text, `Support Us` as the button text
5. In the `Before Footer` widget area add a text widget and add 2 galleries with 3 images via the `Add Media` button. Add an `<hr/>` after each gallery
6. In the `Footer 1` widget area add a `Navigation Menu` widget and select the `Footer Navigation` menu
7. Leave `Footer 2` widget area empty
8. In the `Footer 3` widget area add a search widget
9. In the `Footer 4` widget area add a text widget and paste this `44 Page St. Suite 504, San Francisco, CA 94102 | (415) 495-7377`
10. In the `Footer 4` widget area add a `Largo Follow` widget with no title
11. View on the frontend and verify it all looks ok
